### PR TITLE
[ticket/14334] Use phpbb_get_avatar() in user_loader

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-14334

### CODE REVIEW

File: phpBB/phpbb/user_loader.php [hunk 1]
```diff
@@ -189,12 +189,7 @@ public function get_avatar($user_id, $query = false, $lazy = false)
 			return '';
 		}
 
-		if (!function_exists('get_user_avatar'))
-		{
-			include($this->phpbb_root_path . 'includes/functions_display.' . $this->php_ext);
-		}
-
-		return get_user_avatar($user['user_avatar'], $user['user_avatar_type'], $user['user_avatar_width'], $user['user_avatar_height'], 'USER_AVATAR', false, $lazy);
+		return phpbb_get_avatar(\phpbb\avatar\manager::clean_row($user, 'user'), 'USER_AVATAR', false, $lazy);
```
Comment (VSEphpbb) [inline phpBB/phpbb/user_loader.php]: or maybe just:
`phpbb_get_user_avatar($user, 'USER_AVATAR', false, $lazy);`

Comment (marc1706) [inline phpBB/phpbb/user_loader.php]: I dislike using the wrapper function just for the sake of using the wrapper function. But if you guys like it I can go for it.

Comment (VSEphpbb) [inline phpBB/phpbb/user_loader.php]: @marc1706 learn to love the wrapper :smile: 
giphy

Comment (marc1706) [inline phpBB/phpbb/user_loader.php]: grumpy-cat-no-1

**Pull request merged**